crates/turbopack-ecmascript/src/lib.rs

```diff
@@ -69,6 +69,7 @@ use crate::{
 pub enum EcmascriptModuleAssetType {
     Ecmascript,
     Typescript,
+    TypescriptSource,
```

Comment:
Please add a comment explaining the difference between TS and TSSource. It seems TSSource is code that the dev is writing, where code they're importing?